INSTALL-XAMPP.md

```diff
@@ -0,0 +1,36 @@
+# Install Notes for Windows with XAMPP
+
+It may be handy to install the tools on
+windows for testing and/or developing.
+For such purposes you can install and
+run the tools in cominbation with
+XAMPP.
+
+ 1. Install XAMPP, e.g. in the folder `C:\xampp\`
+ 2. Clone malibu in the `htdocs` subfolder
+ 3. Download jQuery and place it in `.\htdocs\malibu\isbn`
+ 4. Install [`yaz`](http://www.indexdata.com/yaz) for windows.
```

Comment:
+ and make sure that the `bin` directory of `YAZ` is part of the PATH environment variable.